build/CopyFilesToStagingDir.ps1

```diff
@@ -35,93 +35,9 @@ function PublishFile {
 
 PublishFile $OverrideDir\DynamicDependency-Override.json $FullPublishDir\
```

Comment:
this?

Reply:
I don't think so, at least, not directly. It's not failing in "Copy files to staging dir" but in "Publish artifact: Full Nuget (Reunion DLLs)" (2 steps after the copy). I suspect dropped all the file copies to a dir so we don't have the dir now

Yeah. That sounds like it

```
steps:
  - task: powershell@2
    displayName: 'Copy files to staging dir'
    inputs:
      targetType: filePath
      filePath: build\CopyFilesToStagingDir.ps1
      arguments: ...-NugetDir '$(Build.ArtifactStagingDirectory)\FullNuget'...

Maybe we can just mkdir and make him happy. Let's see